docs/UserGuide.adoc

```diff
@@ -249,40 +245,32 @@ image::ug-findcExamplePart2.png[]
 image::ug-findcExamplePart3.png[]
 
 ==== Common errors / problems
-You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them. You will also get a better understanding of the reply from the result box when using `find-c`.
+You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them.
 
 ===== Using incomplete KEYWORD
-You might wonder why `find-c` do not show the clients even though the client is clearly in your client list records. It might be possible that you have entered an incomplete name and does not match any word in the name of that client. The example below might help you understand better.
+You might wonder why `find-c` do not show the clients even though the client is clearly in your client list records. It might be possible that you have entered an incomplete name and does not match any word in the name of that client.
 
-Example:
 You want to find the client named "Bryan Low" in *Client List*. Below shows that "Bryan Low" indeed exists and is recorded.
 
 image::ug-findcIncompleteNamePart1.png[]
 
-Type `find-c bry` into *Command Box*, and press `Enter` to execute it.
+Enter `find-c bry` into *Command Box*. You will obtain "0 clients listed!" in *Result Box*.
```

Comment:
Might want to put the **Result Box** in the beginning: "**Result Box** shows `0 clients listed!`"